text/0000-rfc-process-update.md

```diff
@@ -0,0 +1,166 @@
+[crates.io default ranking](https://github.com/rust-lang/rfcs/blob/master/text/1824-crates.io-default-ranking.md)
+[default URLs for documentation](https://github.com/rust-lang/rfcs/blob/master/text/1826-change-doc-default-urls.md).
+The latter one, for example, would have been a useful RFC when designing the new [API docs](https://github.com/ember-learn/ember-api-docs).
+
+Rust has RFCs for documentation, website, internals, tools.
+We only have emberjs and ember-cli due to them having been implemented separately.
+
+This makes it so that a potential contributor does not know where to submit a proposal.
+
+### Lingering RFCs
+
+The emberjs/rfcs repository currently has [54 issues](https://github.com/emberjs/rfcs/issues) and [62 PRs](https://github.com/emberjs/rfcs/pulls) open, and ember-cli/rfcs has [183 issues](https://github.com/ember-cli/ember-cli/issues) and [49 PRs](https://github.com/ember-cli/ember-cli/pulls).
+A significant percentage of these have not been active in the recent past.
+
+We have kept PRs and issues open so people could more easily find the discussions,
+but this has instead given a negative feeling of staleness, as RFCs linger open without new feedback.
+
+## Detailed design
+
+To address the issues raised in the “Motivation” section, I will propose changes to the current process.
+
+### Implement Core Champion
+
+To make sure that RFCs receive adequate support from the team, Ember CLI has implemented the idea of a champion associated with each RFC.
+The goal is that in seeking a champion from the team,
+the RFC author starts a dialogue with the team and gets some early feedback.
+The champion is then responsible for representing the RFC in team meetings, and for shepherding its progress.
+
+This will be imported into emberjs/rfcs.
+
+### Improve the lifecycle of RFCs
+
+To address the triaging and inactivity problem of RFCs, I introduce the concept of FCP to close.
+
+Closing an RFC should be viewed as another triaging tool, not as a rejection of the RFC.
+Sometimes a rewrite of an RFC would be so fundamental that it would benefit of a fresh discussion in a new thread.
+Sometimes the original author is no longer active (Champions should help here as well),
+and someone else might want to take over the work in a new RFC.
+Sometimes the timing might not be right, or the feature might have been addressed some other way, and yes,
+sometimes it might be something that is not aligned with the team's values for the project.
+
+A good example of this is the [Named Blocks RFC](https://github.com/emberjs/rfcs/blob/master/text/0226-named-blocks.md#motivation),
+which lists in the motivation section previous attempts at similar ideas.
+
+Like the FCP to merge process, once an RFC is marked as FCP to close there will be a period of one week where people can raise new concerns.
+After that period of one week, the respective team will review and close the RFC or extend the period for another week.
+
```

Comment:
I think this is the most likely section people may object to. Try to anticipate the objections and work them in here. i.e. "won't this discourage submitters from participating in the future?"